### ISSUE TITLE
command-line params don't override config

### ISSUE THREAD

**Issue by qrkourier** [clueboard/milc#65]
This works correctly:

```bash
# overrides config
$ mycli --myopt myarg
```

This does not override
```bash
# config overrides command-line
$ mycli --myopt=myarg
```

The only difference seems to be whether space-sep or `=`-sep'd opt=arg syntax is used. I'm investigating and will update this issue if I find the cause.

**Comment by qrkourier** [comment]
```bash
 ❯ pip show milc
Name: milc
Version: 1.6.5
Summary: Opinionated Batteries-Included Python 3 CLI Framework.
Home-page: https://milc.clueboard.co/
Author: skullydazed
Author-email: skullydazed@gmail.com
License: MIT License
Location: /home/kbingham/.pyenv/versions/3.9.0/lib/python3.9/site-packages
Requires: appdirs, argcomplete, colorama, halo, spinners
Required-by: netfoundry

```

**Comment by skullydazed** [comment]
Thanks for the report! I've fixed it in c901c3b, can you give that a try? If that works for you I'll cut a new release.

**Comment by qrkourier** [comment]
Wow! Thank you so much. That was quick, and it's encouraging to see the batteries are _still_ included with MILC. This is the first CLI framework I've worked with and it's been a pleasure.

Confirmed. With that rev I am able to override the general config with wither space or equal sep'd command-line params.

**Comment by skullydazed** [comment]
Glad to hear it! I've released 1.6.6 with that fix.
